### PR TITLE
Clarify selection criteria for GSoC 2024

### PR DESCRIPTION
**Pull request by DubeySandeep**

### CODE REVIEW

File: Google-Summer-of-Code-2024.md [hunk 1]
```diff
@@ -246,11 +246,13 @@ Here's some advice about proposals and milestone timeline planning that we colla
...
 
-In order to select contributors for GSoC, we will mainly be looking at three things (the first of this is the most important, and the other two are of roughly equal weight):
+To select contributors for GSoC, we will evaluate candidates based on a set of criteria designed to ensure we select individuals who not only possess the necessary skills but also demonstrate the ability to contribute effectively to the project. The criteria are as follows, listed in order of significance::
 
-- Whether the contributor has the required skills for the project
-- The quality of the submitted proposal
-- Our prior experience working with the contributor (do they keep commitments, communicate well, demonstrate independence/initiative/responsiveness, help others, etc.)
+- **Primary Criterion: Required Skills for the Project** - This is the most critical factor in our selection process. A contributor must have the necessary skills for the project. Lack of these skills is a deal-breaker and can lead to immediate rejection of the proposal.
+
+- **Secondary Criteria** (of Equal Importance):
```
Comment (seanlip) [inline Google-Summer-of-Code-2024.md]: I think you can change "Equal Importance" to "equal importance"

File: Google-Summer-of-Code-2024.md [hunk 2]
```diff
@@ -246,11 +246,13 @@ Here's some advice about proposals and milestone timeline planning that we colla
...
+    - **Quality of the Submitted Proposal** - The proposal's clarity, thoroughness, feasibility, and alignment with project goals are key. This criterion helps us gauge the applicant's understanding of the project requirements.
```
Comment (seanlip) [inline Google-Summer-of-Code-2024.md]: This criterion helps us gauge the applicant's understanding of the project requirements. The proposal should align with project goals, and be clear, thorough, and feasible.

File: Google-Summer-of-Code-2024.md [hunk 3]
```diff
@@ -246,11 +246,13 @@ Here's some advice about proposals and milestone timeline planning that we colla
...
+    - **Prior Experience Working with the Contributor** - We consider our previous interactions with the contributor, focusing on their reliability, communication skills, independence, initiative, responsiveness, and willingness to assist others. This assessment allows us to predict how well the contributor will integrate with our team and contribute to the project's success.
```
Comment (seanlip) [inline Google-Summer-of-Code-2024.md]: our team --> the Oppia developer community

**Review by seanlip — COMMENTED** [review]
Thanks @DubeySandeep, just a couple suggestions, feel free to merge after fixing. Thanks!

File: Google-Summer-of-Code-2024.md (hunk 1 above)
Comment (DubeySandeep) [reply Google-Summer-of-Code-2024.md]: Done!

File: Google-Summer-of-Code-2024.md (hunk 2 above)
Comment (DubeySandeep) [reply Google-Summer-of-Code-2024.md]: Done!

File: Google-Summer-of-Code-2024.md (hunk 3 above)
Comment (DubeySandeep) [reply Google-Summer-of-Code-2024.md]: Done!

**Review by seanlip — APPROVED** [review]

**Pull request merged**